client/src/components/PreviousTripsCard/PreviousTripsCard.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import PreviousTripsCard from "./PreviousTripsCard";
+import TripCard from "../TripCard/TripCard";
+import Card from "../Card/Card";
+import example1 from "../../../../assets/images/card_small2.png";
+
+storiesOf("PreviousTripsCard", module)
+  .add("Default, Two Trips", () => (
+    <PreviousTripsCard trips={true} link="/">
+      <h1>Previous Trips</h1>
+      <TripCard src={example1} location="Puerto Rico" date="April 2019" />
+      <TripCard src={example1} location="Puerto Rico" date="April 2019" />
```

Comment:
Add a different trip here! I think there is one for Hawaii you can use to make your example look more realistic with different trips.